owl/SOMA-ACT.owl

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns="http://www.ease-crc.org/ont/SOMA.owl#"
-     xml:base="http://www.ease-crc.org/ont/SOMA.owl"
-     xmlns:SOMA="http://www.ease-crc.org/ont/SOMA.owl#"
-     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+<rdf:RDF xmlns="http://www.ease-crc.org/ont/SOMA-ACT.owl#"
+     xml:base="http://www.ease-crc.org/ont/SOMA-ACT.owl"
```

Comment:
Protege has changed back the xml:base :(
How can we avoid that?
It must be that it overwrites it with the ontology rdf:about?

Reply:
I guess most pragmatic and easy would be to change all ontology rdf:about to SOMA.owl

Reply:
this would change all the rdf:about's: https://github.com/ease-crc/soma/pull/97

Reply:
does not seem like an option because Protege would not load two modules next to each other anymore :(

Reply:
ok, change of plan. part of the refactoring must happen in the uglify step